### PR TITLE
update testing to actix-web 2.0.0 and improve stream testing code

### PR DESCRIPTION
**Pull request by estin**
Hi!
I want to improve testing docs for stream response.

I try to show how to get chunk by chunk. In my project I'm test server-sent-event endpoint by fire events on server and check stream behavior.

I can't find any guides how to test on actix-web 2.0.
And this PR may content some wrong ways.

Any way I have success story with actix-web 2.0 and testing in my project )

```
$ cargo check
$ cargo test
```

P.S. Sorry for my poor English

### CODE REVIEW

**Comment by estin** [comment]
I'm don't know why travis check failed...
Trying locally like this without errors

```bash
$ rustup install 1.40.0

$ rustc +1.40.0 --version
rustc 1.40.0 (73528e339 2019-12-16)

$ cargo +1.40.0 --version
cargo 1.40.0 (bc8e4c8be 2019-11-22)

$ cd examples
$ cargo +1.40.0 check
...
    Finished dev [unoptimized + debuginfo] target(s) in 2m 05s
```

**Comment by JohnTitor** [comment]
I guess this PR in unrelated, let me take a look.

**Comment by estin** [comment]
Same issue on Gitter chat  https://gitter.im/actix/actix?at=5e1f8022c86a6f1fb3837b18
And possible solution may be `cargo update`

File: examples/testing/src/stream_response.rs [hunk 1]
```diff
@@ -32,16 +35,42 @@ pub fn main() {
...
+            )
+        );
+
+        // remaining part
+        let bytes = test::load_stream(resp.take_body().into_stream()).await;
         assert!(
-            resp == Bytes::from_static(
-                b"data: 5\n\ndata: 4\n\ndata: 3\n\ndata: 2\n\ndata: 1\n\n"
+            bytes.unwrap() == Bytes::from_static(
+                b"data: 3\n\ndata: 2\n\ndata: 1\n\n"
```
Comment (JohnTitor) [inline examples/testing/src/stream_response.rs]: Could you also change to `assert_eq!` here?

Comment (JohnTitor) [reply examples/testing/src/stream_response.rs]: Other `assert!` in this file are also.

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**